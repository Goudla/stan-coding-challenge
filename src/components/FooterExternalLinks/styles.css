.footer-external-links {
  display: flex;
}

/* @media (max-width: 736px) {
  .footer-external-links {
    display: block;
  }
} */
