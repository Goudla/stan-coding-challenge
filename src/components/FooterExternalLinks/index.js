import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class FooterExternalLinks extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <div className="footer-external-links">
        {children}
      </div>
    );
  }
}

export default FooterExternalLinks;
