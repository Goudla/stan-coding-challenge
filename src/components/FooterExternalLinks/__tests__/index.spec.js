import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import FooterExternalLinks from '../';

describe('<FooterExternalLinks />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<FooterExternalLinks>Test</FooterExternalLinks>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
