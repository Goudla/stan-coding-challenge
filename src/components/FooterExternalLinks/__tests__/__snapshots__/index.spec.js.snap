// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterExternalLinks /> Should render 1`] = `
<div
  className="footer-external-links"
>
  Test
</div>
`;
